plugins/mcreator-localization/help/fr_FR/villagertrades/sale.md

```diff
@@ -0,0 +1 @@
+L'item que tu obtiens en résultat de l'échange.
```

Comment:
To be more constsitent with other tips (in general).
```suggestion
L'item obtenu comme résultat de l'échange.
```

Reply:
Done!